### DISCUSSION TITLE
Stacked Footer

### DISCUSSION THREAD

**Discussion by ainnamran** [Laravel-Backpack/community-forum#1062] [Q&A (Help)]
![image](https://github.com/Laravel-Backpack/community-forum/assets/173441882/da00847d-c50d-4cd8-959c-958e881bbefa)

How can i change the footer to the end the page. The default it stacked on my field. 

`@extends(backpack_view('layouts.plain'))
@section('header')
    <section>
        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
        
    </section>
    <style>
        .card_panel {
            /* padding-top: 50px; */
            padding: 20px;
            border-radius: 8px;
            width: 90%;
            margin: auto;
            min-height: 800px; /* Adjust this as needed */
            background-color: #add8e6; /* Light blue background */
        }

        .bg-light-blue {
            --bs-text-opacity: 1;
            background-color: #AECCE4 !important;
        }

        .row{
            text-align: center;
            justify-content: center;
            /* font-size: 40px; */
            padding-bottom: 50px;
            padding-top: 50px;
        }

        .text-input{
            border: none;
            width: 100%;
        }

        .title{
            font-size: 14px;
        }

        .input-td{
            width: 100%;
        }

        .label-td{
            text-align: center;
            padding: 20px;
        }

        .table-upper{
            width: 100%;
        }

        .special-input-form {
            /* background-color: white; */
            border: none;
            border-radius: 20px;
            font-size:1.5rem;
            width: 50%;
            font-size: 18px;
            /* padding: 10px; */
        }
        
    </style>
    </head>
@endsection
@section('content')
<br/><br/>
<div class="card_panel bg-white bg-light-blue">
    <img class="rounded-circle mx-auto d-block" src="https://i.ibb.co/KLx0syv/MKH-Logo-02.png" alt="MKH logo" id="company_image" height ="15%" width="15%">
    <div class="container">
        <div class="form-group row">
            <form action="{{ route('applySubmit') }}" method="POST">
                @csrf
                <table class = "table-upper">
                    <tr>
                        <td colspan = '6'><b class="third_green"><input type="file" id="avatar" name="avatar" accept="image/*"></b></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Full Name <br/>(as per MyKad) </b></td>
                        <td ><input id="name" name="name" class="text-input third_green" type="text" required></td>
                        <td class = "label-td"><b class="title third_green">Email </b></td>
                        <td><input name="email" id="email" class="text-input third_green" type="text" required></td>
                        <td class = "label-td"><b class="title third_green">MyKad No. </b></td>
                        <td><input name="ic_number" id="ic_number" class="text-input third_green" type="text" required></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Date of Birth </b></td>
                        <td ><input name="dob" id="dob" class="text-input third_green" type="date" required></td>
                        <td class = "label-td"><b class="title third_green">Citizenship </b></td>
                        <td><select name="citizenship" id="citizenship" class="text-input third_green" required>
                            <option value="">Select country</option>
                            <option value="Afghanistan">Afghanistan</option>
                            <option value="Albania">Albania</option>
                            <option value="Algeria">Algeria</option>
                            <option value="Andorra">Andorra</option>
                            <option value="Angola">Angola</option>
                            <option value="Antigua and Barbuda">Antigua and Barbuda</option>
                            <option value="Argentina">Argentina</option>
                            <option value="Armenia">Armenia</option>
                            <option value="Australia">Australia</option>
                            <option value="Austria">Austria</option>
                            <option value="Azerbaijan">Azerbaijan</option>
                            <option value="Bahamas">Bahamas</option>
                            <option value="Bahrain">Bahrain</option>
                            <option value="Bangladesh">Bangladesh</option>
                            <option value="Barbados">Barbados</option>
                            <option value="Belarus">Belarus</option>
                            <option value="Belgium">Belgium</option>
                            <option value="Belize">Belize</option>
                            <option value="Benin">Benin</option>
                            <option value="Bhutan">Bhutan</option>
                            <option value="Bolivia">Bolivia</option>
                            <option value="Bosnia and Herzegovina">Bosnia and Herzegovina</option>
                            <option value="Botswana">Botswana</option>
                            <option value="Brazil">Brazil</option>
                            <option value="Brunei">Brunei</option>
                            <option value="Bulgaria">Bulgaria</option>
                            <option value="Burkina Faso">Burkina Faso</option>
                            <option value="Burundi">Burundi</option>
                            <option value="Côte d Ivoire">Côte d Ivoire</option>
                            <option value="Cabo Verde">Cabo Verde</option>
                            <option value="Cambodia">Cambodia</option>
                            <option value="Cameroon">Cameroon</option>
                            <option value="Canada">Canada</option>
                            <option value="Central African Republic">Central African Republic</option>
                            <option value="Chad">Chad</option>
                            <option value="Chile">Chile</option>
                            <option value="China">China</option>
                            <option value="Colombia">Colombia</option>
                            <option value="Comoros">Comoros</option>
                            <option value="Congo (Congo-Brazzaville)">Congo (Congo-Brazzaville)</option>
                            <option value="Costa Rica">Costa Rica</option>
                            <option value="Croatia">Croatia</option>
                            <option value="Cuba">Cuba</option>
                            <option value="Cyprus">Cyprus</option>
                            <option value="Czechia (Czech Republic)">Czechia (Czech Republic)</option>
                            <option value="Democratic Republic of the Congo">Democratic Republic of the Congo</option>
                            <option value="Denmark">Denmark</option>
                            <option value="Djibouti">Djibouti</option>
                            <option value="Dominica">Dominica</option>
                            <option value="Dominican Republic">Dominican Republic</option>
                            <option value="Ecuador">Ecuador</option>
                            <option value="Egypt">Egypt</option>
                            <option value="El Salvador">El Salvador</option>
                            <option value="Equatorial Guinea">Equatorial Guinea</option>
                            <option value="Eritrea">Eritrea</option>
                            <option value="Estonia">Estonia</option>
                            <option value="Eswatini (fmr. Swaziland)">Eswatini (fmr. Swaziland)</option>
                            <option value="Ethiopia">Ethiopia</option>
                            <option value="Fiji">Fiji</option>
                            <option value="Finland">Finland</option>
                            <option value="France">France</option>
                            <option value="Gabon">Gabon</option>
                            <option value="Gambia">Gambia</option>
                            <option value="Georgia">Georgia</option>
                            <option value="Germany">Germany</option>
                            <option value="Ghana">Ghana</option>
                            <option value="Greece">Greece</option>
                            <option value="Grenada">Grenada</option>
                            <option value="Guatemala">Guatemala</option>
                            <option value="Guinea">Guinea</option>
                            <option value="Guinea-Bissau">Guinea-Bissau</option>
                            <option value="Guyana">Guyana</option>
                            <option value="Haiti">Haiti</option>
                            <option value="Holy See">Holy See</option>
                            <option value="Honduras">Honduras</option>
                            <option value="Hungary">Hungary</option>
                            <option value="Iceland">Iceland</option>
                            <option value="India">India</option>
                            <option value="Indonesia">Indonesia</option>
                            <option value="Iran">Iran</option>
                            <option value="Iraq">Iraq</option>
                            <option value="Ireland">Ireland</option>
                            <option value="Italy">Italy</option>
                            <option value="Jamaica">Jamaica</option>
                            <option value="Japan">Japan</option>
                            <option value="Jordan">Jordan</option>
                            <option value="Kazakhstan">Kazakhstan</option>
                            <option value="Kenya">Kenya</option>
                            <option value="Kiribati">Kiribati</option>
                            <option value="Kuwait">Kuwait</option>
                            <option value="Kyrgyzstan">Kyrgyzstan</option>
                            <option value="Laos">Laos</option>
                            <option value="Latvia">Latvia</option>
                            <option value="Lebanon">Lebanon</option>
                            <option value="Lesotho">Lesotho</option>
                            <option value="Liberia">Liberia</option>
                            <option value="Libya">Libya</option>
                            <option value="Liechtenstein">Liechtenstein</option>
                            <option value="Lithuania">Lithuania</option>
                            <option value="Luxembourg">Luxembourg</option>
                            <option value="Madagascar">Madagascar</option>
                            <option value="Malawi">Malawi</option>
                            <option value="Malaysia">Malaysia</option>
                            <option value="Maldives">Maldives</option>
                            <option value="Mali">Mali</option>
                            <option value="Malta">Malta</option>
                            <option value="Marshall Islands">Marshall Islands</option>
                            <option value="Mauritania">Mauritania</option>
                            <option value="Mauritius">Mauritius</option>
                            <option value="Mexico">Mexico</option>
                            <option value="Micronesia">Micronesia</option>
                            <option value="Moldova">Moldova</option>
                            <option value="Monaco">Monaco</option>
                            <option value="Mongolia">Mongolia</option>
                            <option value="Montenegro">Montenegro</option>
                            <option value="Morocco">Morocco</option>
                            <option value="Mozambique">Mozambique</option>
                            <option value="Myanmar">Myanmar</option>
                            <option value="Namibia">Namibia</option>
                            <option value="Nauru">Nauru</option>
                            <option value="Nepal">Nepal</option>
                            <option value="Netherlands">Netherlands</option>
                            <option value="New Zealand">New Zealand</option>
                            <option value="Nicaragua">Nicaragua</option>
                            <option value="Niger">Niger</option>
                            <option value="Nigeria">Nigeria</option>
                            <option value="North Korea">North Korea</option>
                            <option value="North Macedonia">North Macedonia</option>
                            <option value="Norway">Norway</option>
                            <option value="Oman">Oman</option>
                            <option value="Pakistan">Pakistan</option>
                            <option value="Palau">Palau</option>
                            <option value="Palestine State">Palestine State</option>
                            <option value="Panama">Panama</option>
                            <option value="Papua New Guinea">Papua New Guinea</option>
                            <option value="Paraguay">Paraguay</option>
                            <option value="Peru">Peru</option>
                            <option value="Philippines">Philippines</option>
                            <option value="Poland">Poland</option>
                            <option value="Portugal">Portugal</option>
                            <option value="Qatar">Qatar</option>
                            <option value="Romania">Romania</option>
                            <option value="Russia">Russia</option>
                            <option value="Rwanda">Rwanda</option>
                            <option value="Saint Kitts and Nevis">Saint Kitts and Nevis</option>
                            <option value="Saint Lucia">Saint Lucia</option>
                            <option value="Saint Vincent and the Grenadines">Saint Vincent and the Grenadines</option>
                            <option value="Samoa">Samoa</option>
                            <option value="San Marino">San Marino</option>
                            <option value="Sao Tome and Principe">Sao Tome and Principe</option>
                            <option value="Saudi Arabia">Saudi Arabia</option>
                            <option value="Senegal">Senegal</option>
                            <option value="Serbia">Serbia</option>
                            <option value="Seychelles">Seychelles</option>
                            <option value="Sierra Leone">Sierra Leone</option>
                            <option value="Singapore">Singapore</option>
                            <option value="Slovakia">Slovakia</option>
                            <option value="Slovenia">Slovenia</option>
                            <option value="Solomon Islands">Solomon Islands</option>
                            <option value="Somalia">Somalia</option>
                            <option value="South Africa">South Africa</option>
                            <option value="South Korea">South Korea</option>
                            <option value="South Sudan">South Sudan</option>
                            <option value="Spain">Spain</option>
                            <option value="Sri Lanka">Sri Lanka</option>
                            <option value="Sudan">Sudan</option>
                            <option value="Suriname">Suriname</option>
                            <option value="Sweden">Sweden</option>
                            <option value="Switzerland">Switzerland</option>
                            <option value="Tajikistan">Tajikistan</option>
                            <option value="Tanzania">Tanzania</option>
                            <option value="Thailand">Thailand</option>
                            <option value="Timor-Leste">Timor-Leste</option>
                            <option value="Togo">Togo</option>
                            <option value="Tonga">Tonga</option>
                            <option value="Trinidad and Tobago">Trinidad and Tobago</option>
                            <option value="Tunisia">Tunisia</option>
                            <option value="Turkey">Turkey</option>
                            <option value="Turkmenistan">Turkmenistan</option>
                            <option value="Tuvalu">Tuvalu</option>
                            <option value="Uganda">Uganda</option>
                            <option value="Ukraine">Ukraine</option>
                            <option value="United Arab Emirates">United Arab Emirates</option>
                            <option value="United Kingdom">United Kingdom</option>
                            <option value="United States of America">United States of America</option>
                            <option value="Uruguay">Uruguay</option>
                            <option value="Uzbekistan">Uzbekistan</option>
                            <option value="Vanuatu">Vanuatu</option>
                            <option value="Venezuela">Venezuela</option>
                            <option value="Vietnam">Vietnam</option>
                            <option value="Yemen">Yemen</option>
                            <option value="Zambia">Zambia</option>
                            <option value="Zimbabwe">Zimbabwe</option>
                        </select></td>
                        <td class = "label-td"><b class="title third_green">Gender </b></td>
                        <td><select name = "gender" id="gender" class="text-input third_green" required>
                            <option value="">Select gender</option>
                            <option value="1">Female</option>
                            <option value="2">Male</option>
                            </select>
                        </td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Race </b></td>
                        <td ><select name = "race" id="race" class="text-input third_green" required>
                            <option value="">Select Race</option>
                            <option value="1">Malay</option>
                            <option value="2">Chinese</option>
                            <option value="3">Indian</option>
                        </select></td>
                        <td class = "label-td"><b class="title third_green">Passport </b></td>
                        <td><input name = "passport" id="passport" class="text-input third_green" type="text"></td>
                        <td class = "label-td"><b class="title third_green">Passport Expiry </b></td>
                        <td><input name = "expiry_passport" id="expiry_passport" class="text-input third_green" type="month"></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Status </b></td>
                        <td><select name = "status" id="status" class="text-input third_green" required>
                            <option value="">Select Status</option>
                            <option value="1">Single</option>
                            <option value="2">Married</option>
                            <option value="3">Divorced</option>
                            <option value="4">Widow</option>
                        </select></td>
                        <td class = "label-td"><b class="title third_green">Religion </b></td>
                        <td><select name = "religion" id="religion" class="text-input third_green" required>
                            <option value="">Select Religion</option>
                            <option value="1">Islam</option>
                            <option value="2">Buddhism</option>
                            <option value="3">Hinduism</option>
                            <option value="4">Sikhism</option>
                            <option value="5">Catholicism</option>
                            <option value="6">Judaism</option>
                            <option value="7">Bahai</option>
                            <option value="8">Confucianism</option>
                            <option value="9">Native</option>
                            <option value="10">Taoism</option>
                            <option value="11">Wicca</option>
                            <option value="12">Shinto</option>
                            <option value="13">Eastern Orthodox</option>
                            <option value="14">Roman Religion</option>
                            <option value="15">Paganism</option>
                            </select></td>
                        <td class = "label-td"><b class="title third_green">Contact No. </b></td>
                        <td><input name = "contact_number" id="contact_number" class="text-input third_green" type="number" required></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Home No. </b></td>
                        <td ><input name = "contact_home" id="contact_home" class="text-input third_green" type="number"></td>
                        <td class = "label-td"><b class="title third_green">Weight (kg) </b></td>
                        <td ><input name = "weight" id="weight" class="text-input third_green" type="number" step="0.01"></td>
                        <td class = "label-td"><b class="title third_green">Height (cm) </b></td>
                        <td><input name = "height" id="height" class="text-input third_green" type="number" step="0.01"></td>
                    </tr>
                    <tr>
                        <td colspan = '6'><b class="third_green">Resedential Address </b></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Line 1 </b></td>
                        <td colspan = '2'><input  name = "unit_no" id="unit_no" class="text-input third_green"></td>
                        <td class = "label-td"><b class="title third_green">Line 2 </b></td>
                        <td colspan = '2'><input  name = "house_address" id="house_address" class="text-input third_green"></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Postcode </b></td>
                        <td ><input name = "postcode" id="postcode" class="text-input third_green" type="number"></td>
                        <td class = "label-td"><b class="title third_green">City </b></td>
                        <td><input name = "state" id="city" class="text-input third_green" type="text"></td>
                        <td class = "label-td"><b class="title third_green">State </b></td>
                        <td><input name = "state" id="state" class="text-input third_green" type="text"></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Country </b></td>
                        <td ><input name = "country" id="country" class="text-input third_green" type="text"></td>
                        <td colspan = '4' class = "label-td"><b class="title third_green"></b></td>
                    </tr>
                    <tr>
                        <td colspan = '6'><b class="third_green">Permanent Address </b></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Line 1 </b></td>
                        <td colspan = '2'><input  name = "unit_no1" id="unit_no1" class="text-input third_green"></td>
                        <td class = "label-td"><b class="title third_green">Line 2 </b></td>
                        <td colspan = '2'><input  name = "house_address1" id="house_address1" class="text-input third_green"></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Postcode </b></td>
                        <td ><input name = "postcode1" id="postcode1" class="text-input third_green" type="number"></td>
                        <td class = "label-td"><b class="title third_green">City </b></td>
                        <td><input name = "city1" id="city1" class="text-input third_green" type="text"></td>
                        <td class = "label-td"><b class="title third_green">State </b></td>
                        <td><input name = "state1" id="state1" class="text-input third_green" type="text"></td>
                    </tr>
                    <tr>
                        <td class = "label-td"><b class="title third_green">Country </b></td>
                        <td ><input name = "country1" id="country1" class="text-input third_green" type="text"></td>
                        <td colspan = '4' class = "label-td"><b class="title third_green"></b></td>
                    </tr>
                    <tr>
                        <td colspan = "6">
                            <div id="education-data-group">
                                <b class="title third_green">Education Data</b>
                                <div class="card mb-3 education-group">
                                    <div class="card-body bg-light-blue">
                                        <div class="form-row">
                                            <div class="form-group col-md-3">
                                                <label for="education_from">From</label>
                                                <input type="date" class="form-control" name="education[0][from]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="education_to">To</label>
                                                <input type="date" class="form-control" name="education[0][to]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="education_school">School / College / University</label>
                                                <input type="text" class="form-control" name="education[0][school]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="education_qualification">Qualification</label>
                                                <select class="form-control" name="education[0][qualification]" required>
                                                    <option value="">-</option>
                                                    
                                                </select>
                                            </div>
                                            <div class="form-group col-md-12">
                                                <label for="education_achievements">Achievements</label>
                                                <input type="text" class="form-control" name="education[0][achievements]">
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </td>
                    </tr>
                    <tr>
                        <td><button type="button" class="btn btn-primary mb-3" id="add-education-group">+ Add Group</button></td>
                    </tr>
                    <tr>
                        <td colspan = "6">
                            <div id="employment-history-group">
                                <b class="title third_green">Employment History</b>
                                <div class="card mb-3 employment-group">
                                    <div class="card-body bg-light-blue">
                                        <div class="form-row">
                                            <div class="form-group col-md-3">
                                                <label for="employment_from">From</label>
                                                <input type="date" class="form-control" name="employment[0][from]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="employment_to">To</label>
                                                <input type="date" class="form-control" name="employment[0][to]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="employment_position">Position Held</label>
                                                <input type="text" class="form-control" name="employment[0][position]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="employment_employer">Name of Employer</label>
                                                <input type="text" class="form-control" name="employment[0][employer]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="employment_last_earned">Last Earned</label>
                                                <input type="text" class="form-control" name="employment[0][last_earned]" required>
                                            </div>
                                            <div class="form-group col-md-9">
                                                <label for="employment_reason">Reason For Leaving</label>
                                                <input type="text" class="form-control" name="employment[0][reason]" required>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </td>
                    </tr>
                    <tr>
                        <td><button type="button" class="btn btn-primary mb-3" id="add-employment-group">+ Add Group</button></td>
                    </tr>
                    <tr>
                        <td colspan = "6">
                            <div id="professional-membership-group">
                                <b class="title third_green">Professional Membership / Societies</b>
                                <div class="card mb-3 membership-group">
                                    <div class="card-body bg-light-blue">
                                        <div class="form-row">
                                            <div class="form-group col-md-3">
                                                <label for="membership_from">From</label>
                                                <input type="date" class="form-control" name="membership[0][from]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="membership_to">To</label>
                                                <input type="date" class="form-control" name="membership[0][to]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="membership_position">Position Held</label>
                                                <input type="text" class="form-control" name="membership[0][position]" required>
                                            </div>
                                            <div class="form-group col-md-3">
                                                <label for="membership_society">Professional Bodies / Societies</label>
                                                <input type="text" class="form-control" name="membership[0][society]" required>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </td>
                    </tr>
                    <tr>
                        <td><button type="button" class="btn btn-primary mb-3" id="add-membership-group">+ Add Group</button></td>
                    </tr>
                    <tr>
                        <td colspan = "6">
                            <div id="languages-group">
                                <b class="title third_green">Languages</b>
                                <div class="card mb-3 language-group">
                                    <div class="card-body bg-light-blue">
                                        <div class="form-row">
                                            <div class="form-group col-md-4">
                                                <label for="language_name">Language</label>
                                                <input type="text" class="form-control" name="languages[0][name]" required>
                                            </div>
                                            <div class="form-group col-md-4">
                                                <label for="language_writing">Writing (Level)</label>
                                                <select class="form-control" name="languages[0][writing]" required>
                                                    <option value="">-</option>
                                                    
                                                </select>
                                            </div>
                                            <div class="form-group col-md-4">
                                                <label for="language_spoken">Spoken (Level)</label>
                                                <select class="form-control" name="languages[0][spoken]" required>
                                                    <option value="">-</option>
                                                    
                                                </select>
                                            </div>
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </td>
                    </tr>
                    <tr>
                        <td><button type="button" class="btn btn-primary mb-3" id="add-language-group">+ Add Group</button></td>
                    </tr>
                    <tr>
                        <td colspan = '6'><div id="children-data-group">
                            <b class="title third_green">Children Data</b>
                            <div class="card mb-3">
                                <div class="card-body bg-light-blue">
                                    <div class="form-row">
                                        <div class="form-group col-md-4">
                                            <label for="child_name">Name as in MyKad / MyKid / Passport</label>
                                            <input type="text" class="form-control" id="child_name" name="children[0][name]">
                                        </div>
                                        <div class="form-group col-md-4">
                                            <label for="child_mykad">MyKad / MyKid / Passport</label>
                                            <input type="text" class="form-control" id="child_mykad" name="children[0][mykad]">
                                        </div>
                                        <div class="form-group col-md-2">
                                            <label for="child_gender">Gender M / F</label>
                                            <select class="form-control" id="child_gender" name="children[0][gender]">
                                                <option value="">-</option>
                                                <option value="M">M</option>
                                                <option value="F">F</option>
                                            </select>
                                        </div>
                                        <div class="form-group col-md-2">
                                            <label for="child_dob">Date of Birth</label>
                                            <input type="date" class="form-control" id="child_dob" name="children[0][dob]">
                                        </div>
                                    </div>
                                    <div class="form-row">
                                        <div class="form-group col-md-2">
                                            <label for="child_tax_relief">Tax Relief</label>
                                            <select class="form-control" id="child_tax_relief" name="children[0][tax_relief]">
                                                <option value="">-</option>
                                                <option value="A">A</option>
                                                <option value="B">B</option>
                                                <option value="C">C</option>
                                                <option value="D">D</option>
                                                <option value="E">E</option>
                                                <option value="F">F</option>
                                            </select>
                                        </div>
                                        <div class="form-group col-md-2">
                                            <label for="child_tax_relief_percentage">Tax Relief (%)</label>
                                            <input type="number" class="form-control" id="child_tax_relief_percentage" name="children[0][tax_relief_percentage]">
                                        </div>
                                        <div class="form-group col-md-4">
                                            <label for="child_status">Status (A / B / C / D / E / F)</label>
                                            <input type="text" class="form-control" id="child_status" name="children[0][status]">
                                        </div>
                                    </div>
                                </div>
                            </div>
                        </div></td>
                    </tr>
                    <tr>
                        <td><button type="button" class="btn btn-primary mb-3" id="add-group">+ Add Group</button></td>
                    </tr>
                    <tr>
                        <td colspan = '6'>
                            @include('filter.ageChild')
                        </td>
                    </tr>
                    <tr>
                        <td colspan = '6'><button type="submit" class="btn btn-success btn-submit btn-block">
                            Submit
                        </button></b></td>
                    </tr>
                </table>
            </form>
        </div>
    </div>
</div>
<br/><br/>
@endsection

@section('after_scripts')
    <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/sweetalert2@10/dist/sweetalert2.min.js"></script>
    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/sweetalert2@10/dist/sweetalert2.min.css">
    <script>
    $(document).ready(function() {
            let groupIndex = 1;
            $('#add-group').click(function() {
                let newGroup = `
                <div class="card mb-3">
                    <div class="card-body bg-light-blue">
                        <div class="form-row">
                            <div class="form-group col-md-4">
                                <label for="child_name_${groupIndex}">Name as in MyKad / MyKid / Passport</label>
                                <input type="text" class="form-control" id="child_name_${groupIndex}" name="children[${groupIndex}][name]" required>
                            </div>
                            <div class="form-group col-md-4">
                                <label for="child_mykad_${groupIndex}">MyKad / MyKid / Passport</label>
                                <input type="text" class="form-control" id="child_mykad_${groupIndex}" name="children[${groupIndex}][mykad]" required>
                            </div>
                            <div class="form-group col-md-2">
                                <label for="child_gender_${groupIndex}">Gender M / F</label>
                                <select class="form-control" id="child_gender_${groupIndex}" name="children[${groupIndex}][gender]" required>
                                    <option value="">-</option>
                                    <option value="M">M</option>
                                    <option value="F">F</option>
                                </select>
                            </div>
                            <div class="form-group col-md-2">
                                <label for="child_dob_${groupIndex}">Date of Birth</label>
                                <input type="date" class="form-control" id="child_dob_${groupIndex}" name="children[${groupIndex}][dob]" required>
                            </div>
                        </div>
                        <div class="form-row">
                            <div class="form-group col-md-2">
                                <label for="child_tax_relief_${groupIndex}">Tax Relief</label>
                                <select class="form-control" id="child_tax_relief_${groupIndex}" name="children[${groupIndex}][tax_relief]" required>
                                    <option value="">-</option>
                                    <option value="A">A</option>
                                    <option value="B">B</option>
                                    <option value="C">C</option>
                                    <option value="D">D</option>
                                    <option value="E">E</option>
                                    <option value="F">F</option>
                                </select>
                            </div>
                            <div class="form-group col-md-2">
                                <label for="child_tax_relief_percentage_${groupIndex}">Tax Relief (%)</label>
                                <input type="number" class="form-control" id="child_tax_relief_percentage_${groupIndex}" name="children[${groupIndex}][tax_relief_percentage]" required>
                            </div>
                            <div class="form-group col-md-4">
                                <label for="child_status_${groupIndex}">Status (A / B / C / D / E / F)</label>
                                <input type="text" class="form-control" id="child_status_${groupIndex}" name="children[${groupIndex}][status]" required>
                            </div>
                        </div>
                    </div>
                </div>`;
                $('#children-data-group').append(newGroup);
                groupIndex++;
            });

            let educationIndex = 1;
            $('#add-education-group').click(function() {
                let newGroup = `
                <div class="card mb-3 education-group">
                    <div class="card-body bg-light-blue">
                        <div class="form-row">
                            <div class="form-group col-md-3">
                                <label for="education_from_${educationIndex}">From</label>
                                <input type="date" class="form-control" name="education[${educationIndex}][from]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="education_to_${educationIndex}">To</label>
                                <input type="date" class="form-control" name="education[${educationIndex}][to]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="education_school_${educationIndex}">School / College / University</label>
                                <input type="text" class="form-control" name="education[${educationIndex}][school]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="education_qualification_${educationIndex}">Qualification</label>
                                <select class="form-control" name="education[${educationIndex}][qualification]" required>
                                    <option value="">-</option>
                                    
                                </select>
                            </div>
                            <div class="form-group col-md-12">
                                <label for="education_achievements_${educationIndex}">Achievements</label>
                                <input type="text" class="form-control" name="education[${educationIndex}][achievements]">
                            </div>
                        </div>
                    </div>
                </div>`;
                $('#education-data-group').append(newGroup);
                educationIndex++;
            });

            let employmentIndex = 1;
            $('#add-employment-group').click(function() {
                let newGroup = `
                <div class="card mb-3 employment-group">
                    <div class="card-body bg-light-blue">
                        <div class="form-row">
                            <div class="form-group col-md-3">
                                <label for="employment_from_${employmentIndex}">From</label>
                                <input type="date" class="form-control" name="employment[${employmentIndex}][from]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="employment_to_${employmentIndex}">To</label>
                                <input type="date" class="form-control" name="employment[${employmentIndex}][to]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="employment_position_${employmentIndex}">Position Held</label>
                                <input type="text" class="form-control" name="employment[${employmentIndex}][position]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="employment_employer_${employmentIndex}">Name of Employer</label>
                                <input type="text" class="form-control" name="employment[${employmentIndex}][employer]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="employment_last_earned_${employmentIndex}">Last Earned</label>
                                <input type="text" class="form-control" name="employment[${employmentIndex}][last_earned]" required>
                            </div>
                            <div class="form-group col-md-9">
                                <label for="employment_reason_${employmentIndex}">Reason For Leaving</label>
                                <input type="text" class="form-control" name="employment[${employmentIndex}][reason]" required>
                            </div>
                        </div>
                    </div>
                </div>`;
                $('#employment-history-group').append(newGroup);
                employmentIndex++;
            });

            let membershipIndex = 1;
            $('#add-membership-group').click(function() {
                let newGroup = `
                <div class="card mb-3 membership-group">
                    <div class="card-body bg-light-blue">
                        <div class="form-row">
                            <div class="form-group col-md-3">
                                <label for="membership_from_${membershipIndex}">From</label>
                                <input type="date" class="form-control" name="membership[${membershipIndex}][from]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="membership_to_${membershipIndex}">To</label>
                                <input type="date" class="form-control" name="membership[${membershipIndex}][to]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="membership_position_${membershipIndex}">Position Held</label>
                                <input type="text" class="form-control" name="membership[${membershipIndex}][position]" required>
                            </div>
                            <div class="form-group col-md-3">
                                <label for="membership_society_${membershipIndex}">Professional Bodies / Societies</label>
                                <input type="text" class="form-control" name="membership[${membershipIndex}][society]" required>
                            </div>
                        </div>
                    </div>
                </div>`;
                $('#professional-membership-group').append(newGroup);
                membershipIndex++;
            });

            let languageIndex = 1;
            $('#add-language-group').click(function() {
                let newGroup = `
                <div class="card mb-3 language-group">
                    <div class="card-body bg-light-blue">
                        <div class="form-row">
                            <div class="form-group col-md-4">
                                <label for="language_name_${languageIndex}">Language</label>
                                <input type="text" class="form-control" name="languages[${languageIndex}][name]" required>
                            </div>
                            <div class="form-group col-md-4">
                                <label for="language_writing_${languageIndex}">Writing (Level)</label>
                                <select class="form-control" name="languages[${languageIndex}][writing]" required>
                                    <option value="">-</option>
                                    
                                </select>
                            </div>
                            <div class="form-group col-md-4">
                                <label for="language_spoken_${languageIndex}">Spoken (Level)</label>
                                <select class="form-control" name="languages[${languageIndex}][spoken]" required>
                                    <option value="">-</option>
                                    
                                </select>
                            </div>
                        </div>
                    </div>
                </div>`;
                $('#languages-group').append(newGroup);
                languageIndex++;
            });
        });
    </script>
@endsection`

**Comment by jcastroa87** [comment]
Hello @ainnamran 

I cant see correctly all the code, can you share in other way please.

In any case, you can follow the example on our [footer template](https://github.com/Laravel-Backpack/theme-coreuiv2/blob/main/resources/views/inc/footer.blade.php).

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
